src/imagetransforms/CloudflareImagesTransformer.php

```diff
@@ -25,7 +25,8 @@ public function generateFlexibleVariant(ImageTransform|CloudflareImagesTransform
             $transform[] = "fit={$imageTransform->mode}";
         }
         if ($imageTransform->position) {
-            $transform[] = "gravity={$imageTransform->position}";
+            $gravity = $this->positionToGravity($imageTransform);
+            $transform[] = "gravity={$gravity}";
```

Comment:
```suggestion
            $transform[] = "gravity={$this->positionToGravity($imageTransform)}";
```

Can you check if this work ? I think it does now....

Reply:
This seems to work 
<img width="321" alt="image" src="https://github.com/user-attachments/assets/9c37612e-65e7-4443-a159-8f478a1539a0">

Reply:
It does on my end too! Will adjust.